index.html

```diff
@@ -0,0 +1,1084 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Newsweek</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+</head>
+<body>
+
+    <header class="page-header sticky-top">
+                <div class="container-fluid">
+                    <div class="row header-top pt-4 pb-2">
+                        <div class=" col-md-2 order-md-1 pb-2 pt-2 pl-2">
+                                <div class="date float-left">Wed, Jul 31, 2019</div>
+                        </div>
+                        <div class="col-md-6 order-md-2">
+                            <div class="middle">
+                               <a href="#">
+                                    <img src="../NewsWeek/Img/LOGO-NEWS.jpg" class="img-fluid image-responsive" alt="logo">
```

Comment:
For the image paths, it's better to use "Img/LOGO-NEWS.jpg" directly, call the images relative to the project path.